schema.ts

```diff
@@ -537,40 +537,151 @@ export interface ArraySchema<T extends Schema = Schema> extends BaseSchema {
   items: T;
 }
 
+/**
+ * Fields that may be set on a schema property in the {@link properties} definition
+ * of an object schema.
+ */
 export interface ObjectSchemaProperty {
+  /**
+   * The name of a field in a return value object that should be re-mapped to this property.
+   * This provides a way to rename fields from API responses without writing code.
+   *
+   * Suppose that you're fetching an object from an API that has a property called "duration".
+   * But in your pack, you'd like the value to be called "durationSeconds" to be more precise.
+   * You could write code in your `execute` function to relabel the field, but you could
+   * also use `fromKey` and Coda will do it for you.
+   *
+   * Suppose your `execute` function looked like this:
+   * ```
+   * execute: async function(context) {
+   *   const response = await context.fetcher.fetch({method: "GET", url: "/api/some-entity"});
+   *   // Suppose the body of the response looks like {duration: 123, name: "foo"}.
+   *   return response.body;
+   * }
+   * ```
+   *
+   * You can define your schema like this:
+   * ```
+   * coda.makeObjectSchema({
+   *   properties: {
+   *     name: {type: coda.ValueType.String},
+   *     durationSeconds: {type: coda.ValueType.Number, fromKey: "duration"},
+   *   },
+   * });
+   * ```
+   *
+   * This tells Coda to transform your formula's return value, creating a field "durationSeconds"
+   * whose value comes another field called "duration".
+   */
   fromKey?: string;
+  /**
+   * When true, indicates that an object return value for a formula that has this schema must
+   * include a non-empty value for this property.
+   */
   required?: boolean;
 }
 
+/**
+ * The type of the {@link properties} in the definition of an object schema.
+ * This is essentially a dictionary mapping the name of a property to a schema
+ * definition for that property.
+ */
 export type ObjectSchemaProperties<K extends string = never> = {
   [K2 in K | string]: Schema & ObjectSchemaProperty;
 };
 
+/** @hidden */
 export type GenericObjectSchema = ObjectSchema<string, string>;
 
+// TODO(jonathan, ekoleda): Link to a guide about identities and references.
+/**
+ * An identifier for a schema, allowing other schemas to reference it.
+ *
+ * You may optionally specify an {@link identity} when defining an object schema.
```

Comment:
Some of these links aren't working correctly:

https://coda-packs-sdk--1538.com.readthedocs.build/en/1538/reference/sdk/interfaces/IdentityDefinition/

Should they have been prefixed with a type name?

Reply:
Just checked the same preview link and those aren't working either.